### ISSUE TITLE
Please, remove this repo :(

### ISSUE THREAD

**Issue by tothradoslav** [WINOFFRG/mingo-crack#1]
Hi, @WINOFFRG , I'm the creator of Mingo. I found this crack (which, BTW, is a great technical achievement). 

You might think we are a larger company or a well-funded startup, but no. We are just two guys creating a MongoDB admin for others out there. We have no funding, no management. Everything you see about Mingo is a work of two people. To sustain development, we decided to offer it for a small fee. 

I'd like to ask you, whether you would be willing to remove this repo and thus support Mingo's development, if you like Mingo (as you claim). I'm sure you know how much work it takes to create a product like this, let alone marketing it. We are developers, not marketers. We just want to have an alternative for the current products we don't really prefer...

Thank you for even considering my appeal. 

PS: If you have any ideas for cooperation on Mingo, we'd love to put your knowledge to a greater use :)

**Comment by WINOFFRG** [comment]
Hello @tothradoslav !

I definitely understand how difficult is it for developers to work on such projects and make it a success. I was just waiting for the developer to ask me remove this :) I have no issues at all, I respect your work and will now remove the current release of this crack and will no longer maintain this repo or distribute it via any source. 

I will also add a note on homepage encouraging the people to buy the product from source and support the developer. 

Also, I would be more than happy to share any improvements / bugs that could help this product shine much better.

**Comment by tothradoslav** [comment]
Faith in humanity restored :)

Thank you so much this. What you have done is also a great thing, because we see our weaknesses, possible security issues, etc. 

If there's anything you feel you can contribute, such as ideas, issues, bugs or security problems, please contact me via email info@mingo.io. We are open to any collaboration options!

Thank you again.

**Comment by muhammedfurkan** [comment]
> Faith in humanity restored :)
> 
> Thank you so much this. What you have done is also a great thing, because we see our weaknesses, possible security issues, etc.
> 
> If there's anything you feel you can contribute, such as ideas, issues, bugs or security problems, please contact me via email [info@mingo.io](mailto:info@mingo.io). We are open to any collaboration options!
> 
> Thank you again.

It should be free of charge as long as they are students.

**Comment by tothradoslav** [comment]
yes, however, it's quite difficult to identify / prove being a student, especially in current world, especially dealing with people worldwide. But we are planning on making some changes to our business model. Thanks!
